content/learn/beginner/outputs-1.md

```diff
@@ -0,0 +1,85 @@
++++
+title = "Outputs 1"
+weight = 210
++++
+
+In this chapter, we will go over two ways to output text from a custom command: using the *response*, and later on using
+*template actions*.
+
+## Response
+
+If you paste text into the response field of a custom command, the bot will respond with that text when the command is
+triggered.
+
+For instance, paste the following text into a new custom command:
```

Comment:
"create a new custom command with the following response:"

avoids the duplication of "paste".